src/gui/CMakeLists.txt

```diff
@@ -243,6 +243,8 @@ if (NOT NO_SHIBBOLETH)
     list(APPEND ADDITIONAL_APP_MODULES WebKitWidgets)
```

Comment:
If we are using WebEngine perhaps we should remove the Webkit

Reply:
Yeah once this is done and merged I'll prepare a PR to get rid of webkit